docs/helm-chart-checks.md

```diff
@@ -3,9 +3,10 @@
 Helm chart checks are a set of checks against which the Red Hat Helm chart-verifier tool verifies and validates whether a Helm chart is qualified for a certification from Red Hat. These checks contain metadata that have certain parameters and values with which a Helm chart must comply to be certified by Red Hat. A Red Hat-certified Helm chart qualifies in terms of readiness for distribution in the [Red Hat OpenShift Helm Chart Repository](https://github.com/openshift-helm-charts).
 
 ## Key features
-- You can execute the checks individually. Individual checks can be included or excluded during the verification process using the command line options.
+- You can execute all of the mandatory checks.
+  - Individual checks can be included or excluded during the verification process using the command line options.
 - Each check is independent, and there is no specific execution order.
-- Executing the checks includes running the `helm lint` and `helm template` commands against the chart for Red Hat image certification.
+- Executing the checks includes running the `helm lint`, `helm template`, `helm install`, and `helm test` commands against the chart for Red Hat image certification.
 - The checks are configurable. For example, if a chart requires additional values to be compliant with the checks, configure the values using the options available. The options are similar to those used by the `helm lint` and `helm template` commands.
 - When there are no error messages, the `helm-lint` check passes the verification and is successful. Messages such as `Warning` and `info` do not cause the check to fail.
```

Comment:
Make sure Warning is indeed capitalized. It's your only highlighted text that has been capitalized thus far. Basically, whenever I highlight text in any way, I made sure it's spelt and capitalized exactly as it would be in the software that I'm writing for.

Reply:
Done. All good :)